projects/experimental/directives/card/card.styles.less

```diff
@@ -31,4 +32,83 @@
     [tuiSubtitle] {
         max-width: 100%;
     }
+
+    &[data-size='l'] {
+        --t-size: 100%;
+        height: auto;
+        background: var(--tui-base-01);
```

Comment:
I think we should get rid of background/box-shadow/border-radius — these visual things should come from `tuiSurface`

Reply:
<img width="301" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/75ac3f4f-572a-4af3-8b58-31a355e1e79e">

Reply:
I guess that's a reply for a different comment :) So these are the compensation. But why the selector is so weird? And what's up with the bottom margin? I see there are also right side compensations for different content on the accessories of the header in the specs. Maybe we should at first merge it without any built-in compensation and then try to figure out how we can best address it?

Reply:
fixed